src/main/java/com/glodblock/github/network/CPacketFluidPatternTermBtns.java

```diff
@@ -112,6 +112,9 @@ public IMessage onMessage(CPacketFluidPatternTermBtns message, MessageContext ct
                     case "PatternTerminalEx.ActivePage":
                         cpt.getPatternTerminal().setActivePage(Integer.parseInt(Value));
                         break;
+                    case "PatternTerminalEx.Prioritize":
+                        cpt.getPatternTerminal().setPrioritization(Value.equals("1"));
+                        break;
```

Comment:
extended terminal only?

Reply:
> extended terminal only?

There is no point in making it work in the normal terminal, since this button was created to resolve the bug of sending items in 2 ticks if a pattern is sending more than 9 items (inclusive fluids)